packages/backend-modules/mail/templates/membership_owner_autopay_failed.html

```diff
@@ -50,7 +50,7 @@
                       nicht belasten.
                     </p>
                     <p>
-                      Bitte aktualisieren Sie Ihre Kreditkartendaten {{prolong_url}} oder wählen 
+                      Bitte <a href="{{prolong_url}}">aktualisieren Sie Ihre Kreditkartendaten</a> oder wählen 
```

Comment:
If I remember correctly we need to have links in a separate line in the html for mandrill to parse them correctly to the text version of the mail
```suggestion
                      Bitte 
                      <a href="{{prolong_url}}">aktualisieren Sie Ihre Kreditkartendaten</a>
                       oder wählen 
```